### PR TITLE
chore: Update google analytics to GA4

### PR DESCRIPTION
**Pull request by michaelatmender**
Ticket: MEN-6503
Changelog: None

### CODE REVIEW

**Comment by michaelatmender** [comment]
Also: https://github.com/mendersoftware/saas/pull/994 and 
https://github.com/mendersoftware/mender-kustomize/pull/2

File: src/js/tracking.js [hunk 1]
```diff
@@ -79,9 +79,6 @@ class Tracker {
     } else {
       data = this.currentPageView;
     }
-    if (this.initialized && this.trackingEnabled) {
-      ReactGA.pageview(data);
-    }
```
Comment (mzedel) [inline src/js/tracking.js:84]: based on https://github.com/codler/react-ga4#example this likely has to become:
```
ReactGA.send({ hitType: "pageview", page: data, title: "Some Title" });
```
in order for us to keep tracking pageviews...

**Review by mzedel — CHANGES_REQUESTED** [review]

**Comment by mzedel** [comment]
and the [license checking file](https://github.com/mendersoftware/gui/blob/master/tests/licenses/directDependencies.csv#L81) needs to be adjusted to the MIT license and the `github.com/codler/react-ga4` location

File: src/js/tracking.js (hunk 1 above)
Comment (michaelatmender) [reply src/js/tracking.js:84]: I understood that pageviews should be automatically tracked now, unless we do something custom:
https://developers.google.com/analytics/devguides/migration/measurement/virtual-pageviews#comparison_of_ua_and_ga4_virtual_pageviews
And (the _don't do this_ image) in https://medium.com/@vergil333/google-analytics-4-in-react-a250005c2455

Comment (mzedel) [reply src/js/tracking.js:84]: oh nice! I wasn't aware of this, then sorry about the noise 🙈

**Review by mzedel — APPROVED** [review]

**Pull request merged**